src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -18,72 +20,159 @@ import { FIGMA_URL } from '../../utils/common';
   }}
 />
 
-# Tooltip
+<SectionHeader title={'Tooltip'} />
 
-Generic tooltip that can show content as string
-The background defined is set as property to ThemeProvider with `overrides` property because color for this is global and cannot be changed per case.
-e.g `overrides: { tooltip: { background: { color: 'darkGray', shade: '500' } } }`
-
-## Usage
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
 
-```js
-import { Tooltip } from '@orfium/ictinus';
+## Overview
 
-<Tooltip content="Content you want to show as descrption of tooltip" placement={'right'}>
-  ...
-</Tooltip>;
-```
+Tooltips provide brief, helpful information when a user hovers or focuses on the trigger element.
 
 ## Props
 
 <Props of={Tooltip} />
 
-### Generic Tooltip
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    'To display the full string of a truncated text',
+    'To provide additional context for unlabeled icon buttons',
+    'Use text tooltips to offer additional context that does not support links or actions',
+    'Use interactive tooltips for more complex layouts and to offer simple actions within the tooltip',
+    'Do not display critical information in a tooltip',
+    'Do not use tooltips to display information unrelated to the triggered element',
+  ]}
+/>
+
+## Variants
+
+### Text Tooltip
+
+Text Tooltip takes a string as a content prop. There are 2 variations of Text Tooltips, default and inverted.
+
+<Preview>
+  <Story name="Text Tooltip">
+    <Stack>
+      <Function>
+        {() => {
+          return (
+            <div style={{ padding: '16px' }}>
```

Comment:
on keyboard navigation delay doesnt work as hover. It doesn't mind me at all its even better but it was worth mention it